### PR TITLE
add gravatar image and badge to header, remove logout

### LINKED ISSUE #158: Implement user header section
Remove logout and replace with gravatar. This can be done using the md5 hash of the user's email. Also create a dummy notification circle mirroring the following screenshot:
screen shot 2016-06-17 at 1 43 45 pm

### PR DESCRIPTION
**Pull request by abemassry**
the logout functionality will be replaced in a future PR, when clicking on the user image brings a dropdown and one of the items in there will be logout

Closes #158

### CODE REVIEW

File: scss/manager.scss [hunk 1]
```diff
@@ -198,13 +198,30 @@ a {
...
+            width: 20px;
+            height: 20px;
+            padding-left: 5px;
+            padding-right: 40px;
+            .nav-gravatar-img {
+                width: 40px;
+                height: 40px;
+            }
+            .nav-gravatar-badge {
+                position:absolute;
```
Comment (eatonphil) [inline scss/manager.scss]: We don't really have a scss standard/linter, but could you at least just put a space after the colon?

File: src/reducers/authentication.js [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
       setStorage('authentication/username', action.username);
       setStorage('authentication/email', action.email);
+      setStorage('authentication/email-hash', emailHash);
       setStorage('authentication/scopes', action.scopes);
       return {
         ...state,
         scopes: action.scopes,
         username: action.username,
         email: action.email,
+        emailHash,
```
Comment (eatonphil) [inline src/reducers/authentication.js]: @ddevault isn't there some linter requirement that the unnamed parts of a dict go at some location (first/last/something)?

File: test/components/Navigation.spec.js [hunk 3]
```diff
@@ -6,55 +6,63 @@ import Navigation from '../../src/components/Navigation';
 describe('components/Navigation', () => {
   it('renders username', () => {
     const navigation = mount(
-      <Navigation username={'peanut'} />
+      <Navigation username={'peanut'} email={'peanut@example.com'} />
```
Comment (eatonphil) [inline test/components/Navigation.spec.js]: Isn't your navigation element expecting emailHash not email?

File: src/reducers/authentication.js (hunk 2 above)
Comment (SirCmpwn) [inline src/reducers/authentication.js]: It's @SirCmpwn :)

And yes, but I think they were disabled for some reason or another.

Comment (eatonphil) [inline src/reducers/authentication.js]: Gah. I keep doing that.

Comment (abemassry) [inline src/reducers/authentication.js]: @eatonphil would you like me to move it to the last position?

File: test/components/Navigation.spec.js (hunk 3 above)
Comment (abemassry) [inline test/components/Navigation.spec.js]: Yes that was from my last test when we were md5ing near the display component

**Comment by SirCmpwn** [comment]
LGTM

**Pull request merged**